### PR TITLE
 Cleanup cpaas related unwanted files 

### PR DESCRIPTION
**Pull request by vimalk78**
Cleanup unwanted cpaas files

The scripts are present in https://gitlab.cee.redhat.com/vimalkum/logging-cpaas-scripts,
and are directly copied to operator bundle container repos.

### CODE REVIEW

File: genDockerfile.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+export CI_CONTAINER_VERSION=v4.6.0
+export CI_X_VERSION=4
+export CI_Y_VERSION=6
+export CI_Z_VERSION=0
+
+export CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_COMMIT=$(git rev-parse --verify HEAD)
+export CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_URL=https://github.com/openshift/cluster-logging-operator
+
+#cannot use envsubst here, since replacing images here
```
Comment (alanconway) [inline genDockerfile.sh:9]: Curious - why can't you use envsubst? The sed script is a lot of duplication and easy to make typo bugs.

File: genDockerfile.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    -e "s|\${CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_COMMIT}|$CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_COMMIT|g" \
+    -e "s|\${CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_URL}|$CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_URL|g" \
+    -e "s|openshift/ose-base:rhscl\.ruby\.25\.rhel7|ubi7/ruby-25:latest|g" \
+    -e "s|openshift/golang-builder:1\.14|registry\.svc\.ci\.openshift\.org/openshift/release:golang-1\.13|g" \
+    -e "s|openshift/ose-base:rhscl\.nodejs\.10\.rhel7|ubi7/nodejs-10:latest|g" \
+    -e "s|openshift/ose-base:ubi7|ubi7:latest|g" \
+    -e "s|openshift/ose-base:v4\.6\.0-202007161247\.p0|centos:centos7|g" \
+    Dockerfile.in > Dockerfile.out
+podman build -f ./Dockerfile.out . 
+#$REMOTE_SOURCE/app/elasticsearch
```
[the remark this answers is not in the document]
Comment (vimalk78) [reply genDockerfile.sh]: we dont need these file now

File: genDockerfile.sh (hunk 1 above)
Comment (vimalk78) [reply genDockerfile.sh]: we dont need these file now

**Pull request merged**